tests/integration/integration_test.go

```diff
@@ -183,9 +183,10 @@ var _ = Describe("PCTL", func() {
 				Expect(err).ToNot(HaveOccurred())
 				return strings.Split(string(session), "\n")
 			}
+
 			Eventually(listCmd).Should(ContainElements(
-				"NAMESPACE	NAME  	SOURCE         ",
-				"default  \tmy-sub\tfoo/bar/v0.1.0\t",
+				"NAMESPACE\tNAME                       \tSOURCE                                   \tAVAILABLE UPDATES ",
+				"default  \tlong-name-to-ensure-padding\tfoo/multi-version-weaveworks-nginx/v0.1.0\tv0.1.1           \t",
```

Comment:
gah sorry, I focused on the `long-name-to-ensure-padding` and didn't see the update :joy: